### PR TITLE
Add BOOKSCAN Downloader 2.6.8.

### PR DESCRIPTION
**Pull request by ohkura**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

**Comment by miccal** [comment]
I am not sure if Cask supports Adobe Integrated Runtime installers?

**Comment by reitermarkus** [comment]
@miccal, yes: https://github.com/Homebrew/brew/blob/b86c8efb79b3ed835d552c4d7416640ef10caf21/Library/Homebrew/cask/lib/hbc/container/air.rb

File: Casks/bookscan-downloader.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask 'bookscan-downloader' do
+  version '2.6.8'
+  sha256 'f5c172a02803524fd0dec6b0a1556af5f9248f4eeb35a54caa3a1b3228fd616d'
+
+  url "http://system.bookscan.co.jp/downloader/bookscanDownloader#{version}.air"
+  name 'BOOKSCAN Downloader'
```
Comment (reitermarkus) [inline Casks/bookscan-downloader.rb]: Also add the Japanese name here.

Comment (ohkura) [inline Casks/bookscan-downloader.rb]: Thanks for taking a look.  Added Japanese name.

**Comment by ohkura** [comment]
Thanks!

**Pull request merged**